README.md

```diff
@@ -18,35 +18,70 @@ balena application.
-  -v, --version <VERSION>            Version of balena-os image to download
-  -c, --config <CONFIG_JSON>         Path to balena config.json
-      --log-level <LOG_LEVEL>        Set log level, one of [error,warn,info,debug,trace] [default: info]
-      --log-file <LOG_FILE>          Set stage1 log file name
-      --backup-cfg <BACKUP-CONFIG>   Backup configuration file
-      --s2-log-level <S2_LOG_LEVEL>  Set stage2 log level, one of [error,warn,info,debug,trace]
-      --no-ack                       Scripted mode - no interactive acknowledgement of takeover
-      --pretend                      Pretend mode, do not flash device
-      --stage2                       Internal - stage2 invocation
-      --tar-internal                 Use internal tar instead of external command
-      --no-cleanup                   Debug - do not cleanup after stage1 failure
-      --no-os-check                  Do not check if OS is supported
-      --no-dt-check                  Do not check if the target device type is valid
-      --no-api-check                 Do not check if balena API is available
-      --no-vpn-check                 Do not check if balena VPN is available
-      --no-efi-setup                 Do not setup EFI boot
-      --no-nwmgr-check               Do not check network manager files exist
-      --no-keep-name                 Do not migrate host-name
-  -d, --download-only                Download image only, do not check device and migrate
-      --check-timeout <TIMEOUT>      API/VPN check timeout in seconds.
-  -l, --log-to <LOG_DEVICE>          Write stage2 log to LOG_DEVICE
-  -f, --flash-to <INSTALL_DEVICE>    Use INSTALL_DEVICE to flash balena to
-      --no-wifis                     Do not create network manager configurations for configured wifis
-      --wifi <SSID>                  Create a network manager configuration for configured wifi with SSID
-      --nwmgr-cfg <NWMGR_FILE>       Supply a network manager file to inject into balena-os
-      --change-dt-to <DT_SLUG>       Device Type slug to change to
-  -h, --help                         Print help
+  -w, --work-dir <DIRECTORY>
+          Path to working directory
+  -i, --image <IMAGE>
+          Path to balena-os image
+  -v, --version <VERSION>
+          Version of balena-os image to download
+  -c, --config <CONFIG_JSON>
+          Path to balena config.json
+      --log-level <LOG_LEVEL>
+          Set log level, one of [error,warn,info,debug,trace] [default: info]
+      --log-file <LOG_FILE>
+          Set stage1 log file name
+      --fallback-log
+          Logs to RAM and then dumps logs to balenaOS disk after flashing
+      --fallback-log-filename <FALLBACK_LOG_FILENAME>
+          Set the name of the fallback log [default: fallback.log]
+      --fallback-log-dir <FALLBACK_LOG_DIR>
+          Set the directory name where fallback logs will be persisted on data partition [default: fallback_log]
```

Comment:
When it says "data partition", why would this be in a directory other than /mnt/data/balenahup? Are you referring to situations where the source OS is not balenaOS?

Reply:
the option allows configuring the name of the directory: `/mnt/data/<directory_name>`

perhaps I should remove the "data partition" part?